diff --git a/node_modules/@tanstack/react-query/build/legacy/index.cjs b/node_modules/@tanstack/react-query/build/legacy/index.cjs
index 0d7d36f..3c45dc7 100644
--- a/node_modules/@tanstack/react-query/build/legacy/index.cjs
+++ b/node_modules/@tanstack/react-query/build/legacy/index.cjs
@@ -43,7 +43,8 @@ __export(index_exports, {
   useQueryErrorResetBoundary: () => import_QueryErrorResetBoundary.useQueryErrorResetBoundary,
   useSuspenseInfiniteQuery: () => import_useSuspenseInfiniteQuery.useSuspenseInfiniteQuery,
   useSuspenseQueries: () => import_useSuspenseQueries.useSuspenseQueries,
-  useSuspenseQuery: () => import_useSuspenseQuery.useSuspenseQuery
+  useSuspenseQuery: () => import_useSuspenseQuery.useSuspenseQuery,
+  hashQueryKey: () => hashQueryKey
 });
 module.exports = __toCommonJS(index_exports);
 __reExport(index_exports, require("@tanstack/query-core"), module.exports);
@@ -66,6 +67,8 @@ var import_useMutation = require("./useMutation.cjs");
 var import_mutationOptions = require("./mutationOptions.cjs");
 var import_useInfiniteQuery = require("./useInfiniteQuery.cjs");
 var import_IsRestoringProvider = require("./IsRestoringProvider.cjs");
+var import_query_core2 = require("@tanstack/query-core");
+const hashQueryKey = (queryKey) => import_query_core2.hashKey(queryKey);
 // Annotate the CommonJS export names for ESM import in node:
 0 && (module.exports = {
   HydrationBoundary,
@@ -91,6 +94,7 @@ var import_IsRestoringProvider = require("./IsRestoringProvider.cjs");
   useSuspenseInfiniteQuery,
   useSuspenseQueries,
   useSuspenseQuery,
+  hashQueryKey,
   ...require("@tanstack/query-core"),
   ...require("./types.cjs")
 });
diff --git a/node_modules/@tanstack/react-query/build/legacy/index.d.ts b/node_modules/@tanstack/react-query/build/legacy/index.d.ts
index 45dab83..eb8fd94 100644
--- a/node_modules/@tanstack/react-query/build/legacy/index.d.ts
+++ b/node_modules/@tanstack/react-query/build/legacy/index.d.ts
@@ -18,5 +18,6 @@ export { useMutation } from './useMutation.js';
 export { mutationOptions } from './mutationOptions.js';
 export { useInfiniteQuery } from './useInfiniteQuery.js';
 export { IsRestoringProvider, useIsRestoring } from './IsRestoringProvider.js';
+export declare const hashQueryKey: (queryKey: import('@tanstack/query-core').QueryKey) => string;
 import 'react';
 import 'react/jsx-runtime';
diff --git a/node_modules/@tanstack/react-query/build/legacy/index.js b/node_modules/@tanstack/react-query/build/legacy/index.js
index 66ddb26..dc46a6c 100644
--- a/node_modules/@tanstack/react-query/build/legacy/index.js
+++ b/node_modules/@tanstack/react-query/build/legacy/index.js
@@ -1,6 +1,9 @@
 // src/index.ts
 export * from "@tanstack/query-core";
 export * from "./types.js";
+import { hashKey } from "@tanstack/query-core";
+const hashQueryKey = (queryKey) => hashKey(queryKey);
+export { hashQueryKey };
 import { useQueries } from "./useQueries.js";
 import { useQuery } from "./useQuery.js";
 import { useSuspenseQuery } from "./useSuspenseQuery.js";
diff --git a/node_modules/@tanstack/react-query/build/modern/index.cjs b/node_modules/@tanstack/react-query/build/modern/index.cjs
index 0d7d36f..fb5bc47 100644
--- a/node_modules/@tanstack/react-query/build/modern/index.cjs
+++ b/node_modules/@tanstack/react-query/build/modern/index.cjs
@@ -43,7 +43,8 @@ __export(index_exports, {
   useQueryErrorResetBoundary: () => import_QueryErrorResetBoundary.useQueryErrorResetBoundary,
   useSuspenseInfiniteQuery: () => import_useSuspenseInfiniteQuery.useSuspenseInfiniteQuery,
   useSuspenseQueries: () => import_useSuspenseQueries.useSuspenseQueries,
-  useSuspenseQuery: () => import_useSuspenseQuery.useSuspenseQuery
+  useSuspenseQuery: () => import_useSuspenseQuery.useSuspenseQuery,
+  hashQueryKey: () => hashQueryKey
 });
 module.exports = __toCommonJS(index_exports);
 __reExport(index_exports, require("@tanstack/query-core"), module.exports);
@@ -66,6 +67,8 @@ var import_useMutation = require("./useMutation.cjs");
 var import_mutationOptions = require("./mutationOptions.cjs");
 var import_useInfiniteQuery = require("./useInfiniteQuery.cjs");
 var import_IsRestoringProvider = require("./IsRestoringProvider.cjs");
+var import_query_core2 = require("@tanstack/query-core");
+const hashQueryKey = (queryKey) => import_query_core2.hashKey(queryKey);
 // Annotate the CommonJS export names for ESM import in node:
 0 && (module.exports = {
   HydrationBoundary,
@@ -91,6 +94,7 @@ var import_IsRestoringProvider = require("./IsRestoringProvider.cjs");
   useSuspenseInfiniteQuery,
   useSuspenseQueries,
   useSuspenseQuery,
+  hashQueryKey,
   ...require("@tanstack/query-core"),
   ...require("./types.cjs")
 });
diff --git a/node_modules/@tanstack/react-query/build/modern/index.d.ts b/node_modules/@tanstack/react-query/build/modern/index.d.ts
index 45dab83..eb8fd94 100644
--- a/node_modules/@tanstack/react-query/build/modern/index.d.ts
+++ b/node_modules/@tanstack/react-query/build/modern/index.d.ts
@@ -18,5 +18,6 @@ export { useMutation } from './useMutation.js';
 export { mutationOptions } from './mutationOptions.js';
 export { useInfiniteQuery } from './useInfiniteQuery.js';
 export { IsRestoringProvider, useIsRestoring } from './IsRestoringProvider.js';
+export declare const hashQueryKey: (queryKey: import('@tanstack/query-core').QueryKey) => string;
 import 'react';
 import 'react/jsx-runtime';
diff --git a/node_modules/@tanstack/react-query/build/modern/index.js b/node_modules/@tanstack/react-query/build/modern/index.js
index 66ddb26..dc46a6c 100644
--- a/node_modules/@tanstack/react-query/build/modern/index.js
+++ b/node_modules/@tanstack/react-query/build/modern/index.js
@@ -1,6 +1,9 @@
 // src/index.ts
 export * from "@tanstack/query-core";
 export * from "./types.js";
+import { hashKey } from "@tanstack/query-core";
+const hashQueryKey = (queryKey) => hashKey(queryKey);
+export { hashQueryKey };
 import { useQueries } from "./useQueries.js";
 import { useQuery } from "./useQuery.js";
 import { useSuspenseQuery } from "./useSuspenseQuery.js";
