diff --git a/node_modules/@trpc/react-query/dist/createHooksInternal-063195fc.mjs b/node_modules/@trpc/react-query/dist/createHooksInternal-063195fc.mjs
index 667b33f..00a89a5 100644
--- a/node_modules/@trpc/react-query/dist/createHooksInternal-063195fc.mjs
+++ b/node_modules/@trpc/react-query/dist/createHooksInternal-063195fc.mjs
@@ -361,7 +361,7 @@ function getClientArgs(pathAndInput, opts) {
             throw new Error('Unable to retrieve application context. Did you forget to wrap your App inside `withTRPC` HoC?');
         }
         const { abortOnUnmount , client , ssrState , queryClient , prefetchQuery  } = context;
-        const defaultOpts = queryClient.getQueryDefaults(getArrayQueryKey(pathAndInput, 'query'));
+        const defaultOpts = queryClient.getQueryDefaults?.(getArrayQueryKey(pathAndInput, 'query'));
         if (typeof window === 'undefined' && ssrState === 'prepass' && opts?.trpc?.ssr !== false && (opts?.enabled ?? defaultOpts?.enabled) !== false && !queryClient.getQueryCache().find(getArrayQueryKey(pathAndInput, 'query'))) {
             void prefetchQuery(pathAndInput, opts);
         }
@@ -399,7 +399,7 @@ function getClientArgs(pathAndInput, opts) {
             context: ReactQueryContext
         });
         const actualPath = Array.isArray(path) ? path[0] : path;
-        const defaultOpts = queryClient.getMutationDefaults([
+        const defaultOpts = queryClient.getMutationDefaults?.([
             actualPath.split('.')
         ]);
         const hook = useMutation({
@@ -471,7 +471,7 @@ function getClientArgs(pathAndInput, opts) {
     function useInfiniteQuery$1(pathAndInput, opts) {
         const [path, input] = pathAndInput;
         const { client , ssrState , prefetchInfiniteQuery , queryClient , abortOnUnmount ,  } = useContext();
-        const defaultOpts = queryClient.getQueryDefaults(getArrayQueryKey(pathAndInput, 'infinite'));
+        const defaultOpts = queryClient.getQueryDefaults?.(getArrayQueryKey(pathAndInput, 'infinite'));
         if (typeof window === 'undefined' && ssrState === 'prepass' && opts?.trpc?.ssr !== false && (opts?.enabled ?? defaultOpts?.enabled) !== false && !queryClient.getQueryCache().find(getArrayQueryKey(pathAndInput, 'infinite'))) {
             void prefetchInfiniteQuery(pathAndInput, {
                 ...defaultOpts,
diff --git a/node_modules/@trpc/react-query/dist/createHooksInternal-75aa31d9.js b/node_modules/@trpc/react-query/dist/createHooksInternal-75aa31d9.js
index a62ebbb..b696199 100644
--- a/node_modules/@trpc/react-query/dist/createHooksInternal-75aa31d9.js
+++ b/node_modules/@trpc/react-query/dist/createHooksInternal-75aa31d9.js
@@ -300,7 +300,9 @@ function createRootHooks(config) {
             throw new Error('Unable to retrieve application context. Did you forget to wrap your App inside `withTRPC` HoC?');
         }
         const { abortOnUnmount, client, ssrState, queryClient, prefetchQuery } = context;
-        const defaultOpts = queryClient.getQueryDefaults(getArrayQueryKey(pathAndInput, 'query'));
+        const defaultOpts = typeof queryClient.getQueryDefaults === 'function'
+            ? queryClient.getQueryDefaults(getArrayQueryKey(pathAndInput, 'query'))
+            : undefined;
         if (typeof window === 'undefined' &&
             ssrState === 'prepass' &&
             opts?.trpc?.ssr !== false &&
@@ -341,9 +343,11 @@ function createRootHooks(config) {
         const { client } = useContext();
         const queryClient = useQueryClient({ context: ReactQueryContext });
         const actualPath = Array.isArray(path) ? path[0] : path;
-        const defaultOpts = queryClient.getMutationDefaults([
-            actualPath.split('.'),
-        ]);
+        const defaultOpts = typeof queryClient.getMutationDefaults === 'function'
+            ? queryClient.getMutationDefaults([
+                actualPath.split('.'),
+            ])
+            : undefined;
         const hook = useMutation({
             ...opts,
             mutationKey: [actualPath.split('.')],
@@ -406,7 +410,9 @@ function createRootHooks(config) {
     function useInfiniteQuery$1(pathAndInput, opts) {
         const [path, input] = pathAndInput;
         const { client, ssrState, prefetchInfiniteQuery, queryClient, abortOnUnmount, } = useContext();
-        const defaultOpts = queryClient.getQueryDefaults(getArrayQueryKey(pathAndInput, 'infinite'));
+        const defaultOpts = typeof queryClient.getQueryDefaults === 'function'
+            ? queryClient.getQueryDefaults(getArrayQueryKey(pathAndInput, 'infinite'))
+            : undefined;
         if (typeof window === 'undefined' &&
             ssrState === 'prepass' &&
             opts?.trpc?.ssr !== false &&
